### DISCUSSION TITLE
新版本无法授权其它设备

### DISCUSSION THREAD

**Discussion by WangZaili** [getlantern/lantern-forum-cn#370] [Q&A]
升级到7.8,3后，“账号”下拉菜单里没有了之前的选项“Pro授权设备”，并且也找不到现在能从哪里完成授权其它设备。
从“账户管理”里删除了之前授权过的设备，保证不是因授权量已满额后，依然没有找到能够授权新设备的入口。
请帮忙解答，谢谢！

**Comment by MaxMKMax** [comment]
@WangZaili 您好, 请问您使用iOS系统吗? 如果iOS的话我们暂时没有授权入口. 要等下一个版本呢. 
之后我们给你们补偿延长专业版本时间.

非常抱歉因为系统上出了一些问题, 我们正在修复了. 
iOS以外的, 我们暂时会通过手动帮您升级或发送激活码. 请您提供我您的账号邮箱.
非常抱歉也谢谢您支持蓝灯!

**Reply by BRUCELVCHINA** [reply]
我的邮箱是12011272@qq.com，我现在已经在mac上购买了专业版，但是不知道如何授权IOS手机和IPad如何使用专业版。

**Comment by MaxMKMax** [comment]
@WangZaili 
7.8.4 今晚或明天就会出来了. 也会解决掉之前的问题.

**Comment by WangZaili** [comment]
@Mike777Khoo 感谢您的帮助，我用的是win10系统。不过我刚刚尝试升级到了preview 7.8.5版，菜单和上次发帖时说的7.8,3一样，还是没有授权新设备的入口。
我的账号是relive_w@126.com, 可否帮忙从后台确认下我的账号状态，并且提供一下授权一台新windows设备的激活码？谢谢。

**Comment by paderewsky** [comment]
确实是，我的是电脑端和手机端。电脑系统重新安装后，手机端无法授权电脑端6位验证码，填写了邮件也无法发送。

**Reply by paderewsky** [reply]
我的注册邮箱是aurelius@163.com

**Reply by MaxMKMax** [reply]
@paderewsky 您好! 可以的我让客服帮您恢复状态. 请您提供我购买单据. 然后他们会手动发激活码给您. 请您过几天留有邮箱或GitHub. 谢谢!

**Reply by paderewsky** [reply]
您好，因为时间也久了，我查了支付记录，一时没找到。请问您那有购买日期吗？谢谢

**Reply by MaxMKMax** [reply]
@paderewsky 
您在邮箱里查看Lantern的名字. 应该会收到一封邮件里有购买单据的.

**Reply by paderewsky** [reply]
你好，因为购买最起码是1年左右时间，有些邮件都被删除了。你看有没有什么办法解决的。也不会无中生有的来乱说自己是专业版的。都是付过费的。你们后台应该查到到我这个账号的真实情况。

**Reply by paderewsky** [reply]
@Mike777Khoo 目前用手机授权，有六位数授权码，但是PC端，没有显示对应的6个框，反而也显示了授权码。这个怎么办？

**Reply by MaxMKMax** [reply]
@paderewsky 您好. 请问您有下载最新版本吗? 还有遇到您之前的问题吗? 如果有的话, 请帮我提交个报告. 然后给我票号. 我来帮您反馈问题上去修复. 谢谢您!

**Comment by WangZaili** [comment]
您好，我查了下邮箱，在最近一次续费的时间附近没有找到升级pro成功通知或是电子收据之类的邮件，应该是没有给我发送过吧。您那边根据我上边提供的账号是可以查到账号状态的吧？或者我还想到能提供的就是当时支付的信用卡号了，这个有用吗？有用的话请留个邮箱给我吧，我发过去

**Reply by MaxMKMax** [reply]
@WangZaili 嗯嗯好的. 我帮您向付款团队拿激活码. 但是需要您提供我那个账号邮箱和支付单据. 谢谢.


可以发给我: mike@getlantern.org

**Reply by WangZaili** [reply]
现在的问题就是我这边找不到付款时给过什么单据。当初如果确实发过，这种邮件我是不会删除的。我最近一次续费是大约2年前，那时是不是系统还不会发这个单据？我现在提供我的账号 relive_w@126.com ，从后台应该很容易验证我是付费的pro版用户吧？

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/80a260cb-75ca-4804-b9fb-145af638f6fe)

@WangZaili 您好. 您是专业版了. 支付团队说您只需要重新授权就可以了. 

您可以参考这里怎么授权:
https://lantern.io/zh-Hans/faq

**Comment by Capel-Allen** [comment]
我是今天新买的专业版，但我没法给其它两台新设备关联用啊。找不到那个授权页面

**Reply by MaxMKMax** [reply]
@Capel-Allen 您好. 您可以参考这里怎么授权:
https://lantern.io/zh-Hans/faq

**Comment by Capel-Allen** [comment]
现在是我没法授权在其他设备登录啊， ，怎么同时在3台设备登录使用

Thanks & Best regards


Allen Shi / 石教才
Capel Technology Co.,Ltd
Mobile/WeChat:86-15817325398
E-mail/Skype: ***@***.***
http://www.capelfpc.com/
UL| ISO14001 | ISO9001| IATF16949
Address: Building 5,TaiFeng Industrial Zone, No.10 JianAn road,ShaJing ,BaoAn  ,ShenZhen, China
 
From: Mike K
Date: 2024-04-10 23:54
To: getlantern/lantern-forum-cn
CC: Capel-Allen; Mention
Subject: Re: [getlantern/lantern-forum-cn] 新版本无法授权其它设备 (Discussion #370)
image.png (view on web)
@WangZaili 您好. 您是专业版了. 支付团队说您只需要重新授权就可以了.
您可以参考这里怎么授权:
https://lantern.io/zh-Hans/faq

**Reply by MaxMKMax** [reply]
@Capel-Allen 您好. 请问您的其他两个设备是什么系统和什么版本? IOS xxx? MacOS xxx? Android xxx? Window xxx?

**Comment by WangZaili** [comment]
Hi Mike,
您指的是教程的这里对吧？
![image](https://github.com/getlantern/lantern-forum-cn/assets/16510373/910decdf-821a-48e0-96de-9ccef2658553)
但是我的问题是，现在升级到最近的版本后，下拉菜单里没有教程里说的“Pro授权设备"选项，所以我现在没法给其它设备授权
![image](https://github.com/getlantern/lantern-forum-cn/assets/16510373/5c7dabf0-53f4-4ce4-8f8e-b3a96570635f)
而且我以前是给自己另一台电脑授过权的，所以看上去是新版本里这个功能缺失了，至少是功能入口不见了

**Reply by WangZaili** [reply]
现在菜单里”授权设备“那个选项是**被授权机器**上用的，点进去是输入pro设备上生成的授权码的

**Reply by MaxMKMax** [reply]
@WangZaili 
嗯嗯. 对的. 授权入口应该只有iOS没有. 您是在哪一个设备看到这个授权入口不见了?

**Reply by WangZaili** [reply]
是win10系统。麻烦再看一下最上边我的主帖，里边描述还是比较清楚的

**Reply by MaxMKMax** [reply]
@WangZaili 嗯嗯好的. 我帮您反馈. 您也试试看重新下载最新版本 8.0.0 . 听说都恢复正常了. 

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by WangZaili** [reply]
最新的8.0.0从哪里下载呀？从github上下载目前最新的还是7.8.4和beta 7.8.5

**Reply by WangZaili** [reply]
已提交ticket 71469819 @Mike777Khoo

**Reply by MaxMKMax** [reply]
@WangZaili 我们上周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）。

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Reply by WangZaili** [reply]
用你给的链接下载安装win版本的，启动后 进“设置”里，看到版本号是7.8.5(20240404.173233) 不是8.0.0

**Reply by MaxMKMax** [reply]
@WangZaili 您好. 我正在帮您查问了. 照道理应该是8.0.0版本. 请问您使用什么设备? 比如: Window 7? Window 10?

**Reply by MaxMKMax** [reply]
https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe
@WangZaili

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/ba2ae8fb-6a89-4587-a95a-cae8527defe0)
@WangZaili 那个链接是对的. 如果您看到这个页面, 那就是对的版本了. 8.0.0

**Comment by MaxMKMax** [comment]
@WangZaili https://github.com/getlantern/lantern
您到这个连接照看对应的版本.

**Reply by WangZaili** [reply]
我是win10系统，是按照您上边截图的github repo里的下载页面，点Windows beta那里的Windows 10+链接下载的安装包，并且安装包的链接也是您上边贴的那个。但现在下载完安装后，启用后从蓝灯首页的“设置”点进去，右下角显示的版本号就是我上边说的7.8.5啊。。。
好奇怪。要不您能否把8.0.0的安装包再单独发个附件出来

**Reply by MaxMKMax** [reply]
https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe


@WangZaili 您好. 这个就是 8.0.0 的链接了. 会不会是他们忘了改名字啊? 但很多人都下载成功了.

**Reply by MaxMKMax** [reply]
@WangZaili 如果您的页面看到好像我分享你的图的话. 那就是8.0.0了.

**Reply by WangZaili** [reply]
别人是怎么通过这个链接下载到8.0.0的我不知道，我确实是从上周到刚才下载安装了N次，装完一看，都是7.8.5啊...
所以能否麻烦把您那边已经验证是8.0.0版本的安装文件，以附件形式帮我贴在这里，让我再试一下

**Reply by WangZaili** [reply]
您再跟技术团队对一下release install package吧，我已经看到有其它用户是跟我一样的发现了，下载的.exe安装完是7.8.5 https://github.com/getlantern/lantern-forum-cn/discussions/388#discussioncomment-9186575

**Comment by xrileyx** [comment]
我也遇到了同样的问题。账号登陆后下拉菜单里选项“授权设备”后没有生成授权码，而是要我输入授权码，也找不到现在能从哪里完成授权其它设备。我用的是mac，7.8.4，刚刚下了 桌面应用程序测试版（8.0.0 版本）重新登陆了也没有授权码的界面 @Mike777Khoo

**Reply by MaxMKMax** [reply]
@xrileyx 您好. 已经 7.8.5 很久了. 现在好像已经 7.8.6 了. 您重新下载看看.

然后新的 Flutter 桌面应用程序测试版（8.0.0 版本）。

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Comment by Faker009** [comment]
1. 安装问题：在kubuntu上安装了lantern.deb无法正常启动，经查询是被安装到了/usr/share目录下面，将liblantern.so拷贝至/usr/lib下面才能能启动。

2. pro版本授权问题：无法完成pro版授权，在其他客户端上输入pin码显示成功，回到kubuntu系统打开还是不行，通过邮件账户输入，邮箱收不到pin码

有客服售后吗？

**Reply by MaxMKMax** [reply]
@Faker009 您好, 我们有售后服务的. 请您提供我账号邮箱和支付单据. 我帮您提交问题上去. 请您放心, 我们会帮您解决问题. 谢谢您支持蓝灯!

**Comment by xrileyx** [comment]
Hi there,

Please remove me from this email chain. Thanks!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by MaxMKMax** [comment]
@Faker009  您好, 请您提供我账号邮箱和支付单据. 我帮您提交问题上去. 请您放心, 我们会帮您解决问题. 谢谢您支持蓝灯!
